### PR TITLE
feat(aliases): add scripts for postgres

### PR DESCRIPTION
**Pull request by Jl14Salvador**
- add scripts for postgres, support pqcli module

### CODE REVIEW

File: aliases [hunk 1]
```diff
@@ -8,10 +8,16 @@ alias v="$VISUAL"
...
 alias av='autovance'
-
 alias copy-staging='autovance && ./onboarding/scripts/copy_psql.rb ${MAIN_PRODUCTION} ${MAIN_STAGING} -i api_key -e ./.expire_tables -v && cd ./database-main && g co master && g pull && npm run db-stage up'
 alias r2='r2d2'
 
+#postgres scripts
+alias pg='pgcli' || 'psql'
+alias pg-local='pgcli $MAIN_LOCAL' || 'psql $MAIN_LOCAL'
+alias pg-staging='pgcli $MAIN_STAGING' || 'psql $MAIN_STAGING'
+alias pg-prod='pgcli $MAIN_PRODUCTION' || 'psql $MAIN_PRODUCTION'
```
Comment (forstermatth) [inline aliases]: does this `||` syntax pick whichever one is available?

Comment (Jl14Salvador) [reply aliases]: @forstermatth  that is the idea, but i'm actually having issues just trying to get the alias to work, even without  ||,
I've ran rcup after making the change but it still doesn't recognize the command pg, wasn't sure if there was something else that i needed to do

Comment (trs) [reply aliases]: `source aliases`

**Comment by Jl14Salvador** [comment]
@trs @forstermatth  This syntax works, will call pgcli if it is an available command and if not, will call the postgres command

File: zsh/functions/call-pg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+call-pg() 
+{
+    if type "pgcli" &> /dev/null; then
+        pgcli $1
```
Comment (trs) [inline zsh/functions/call-pg]: Change `$1` to `"$@"`. That way you can pass in multiple arguments to `pg` if needed.

See: https://stackoverflow.com/questions/4824590/propagate-all-arguments-in-a-bash-shell-script

**Pull request merged**